notes/authenc.tex

```diff
@@ -142,4 +143,117 @@ \section{Authenticated Encryption}
+Paterson, Albrecht et al....).\scribenote{Cite all of these.}
+
+% -------------------
+\subsection{Authenticated encryption security.}
+
+Our encryption algorithms don't have confidentiality or intregity guarantees in the face of active attacks. In order to remedy this, we need new schemes.
+
+We will show a first variant of this as a way to build authenticated encryption security.
+Let's begin with an encryption oracle or random bits oracle (ROR).
+Now we add a decryption oracle to which you can submit ciphertexts.
+Recall ROR-CCA1 and ROR-CCA0 from \thref{theorem:ror-cpa}.
+2:50pm curly C is a set, regular C is a string. In ROR-CC0 (the ideal world), decryption always outputs an error symbol. There is no way to produce a ciphertext by mauling. CBC mode will not be secure under this motion. Why must there be a restriction in our first variant that the attacker can't send to dec that you get from enc? Then you'd have to add more logic about what ciphertexts are returned (table of decryptions). If not in table, return bottom. These definitions can be shown equivalent pretty straightforwardly.
+As an aside (to scribe notes) 2:52pm, there's an all-in-one definition. (Leave it to notes.) \scribenote{Address this.}
+
+Our new notion of cihertext integrity just focuses on the ability to build encryption that doesn't decrypt to bottom. Refer to the CTXT game in \thref{theorem:ror-cpa}.\scribenote{Fix this ref or put CTXT in new ref.} We have an encryption and decryption oracle. The goal of an adversary is to try to get \textsf{dec} to return something other than error. %Turns out that ROR-CCA
+It is easy to prove ROR+CTXT. Variant of ROR-CCA which in this game could return...\scribenote{Why?}
+
+Let's see about building ROR-CCA using OCB mode.
+Recall the tweakable block ciphers (OCB-CCA) scheme. Per-message randomness. (2:58pm). Not CTXT secure.
+Example of how we could craft a message trivially to decrypt into something valid? A CTXT attack would be: any ciphertext string of $0^{4n}$ and it would decrypt. No error in any case unless it's the wrong length.
+
+We need redundancy in encryption to reject good vs bad decryptions for a message. Idea (3:01pm). Encode $0^n / 2$. Make the messages smaller blocks and then pad out the blocks... Is this secure? If you query random ciphertexts, could forge decryption in $q^{3n} / 2$. Bounds might be okay...but bounds could prove to be an issue.
+Query until you get a valid padding. Better than birthday bound.
+Per-block of encryption, only processing half as many bits of plaintext.
+%Other idea: add XORed random number in M1 (3:06pm). Might not work.
+Cute trick that people with came up with OCB realized (3:08pm). Add another block... \scribenote{Add this.}
+
+Tweak on a tag value needs to be different from other tweaks. This suffices to provide integrity. Just running the block cipher one more time. This can all be done in one pass. Rather than encrypting in one, and then authenticating.
```

Comment:
a lot of the text looks disjointed and incomplete, but i think based on the `scribenote`s you have already noted that so i won't note them again